packages/rfw/lib/src/dart/text.dart

```diff
@@ -2127,7 +2003,7 @@ class _Parser {
   void _advance() {
     assert(_source.current is! _EofToken);
     final bool advanced = _source.moveNext();
-    assert(advanced == true); // see https://github.com/dart-lang/sdk/issues/47017
+    assert(advanced); // see https://github.com/dart-lang/sdk/issues/47017
```

Comment:
The `== true` was there on purpose. See the issue link.

Reply:
The issue is marked as fixed, though. Should we leave the `== true` out and remove the comment as well or do you prefer to still keep the `== true`?

Reply:
It looks like it was fixed for real, so we should remove the link to the issue then.